user/user.go

```diff
@@ -16,6 +16,7 @@ import (
 const (
 	TimestampFormat      = "2006-01-02T15:04:05-07:00"
 	custodialEmailFormat = "unclaimed-custodial-automation+%020d@tidepool.org"
+	RoleCareTeamMember   = "care_team_member"
```

Comment:
```suggestion
	RoleCareTeamMember   = "care_partner"
```
To match https://github.com/tidepool-org/terraform-modules/pull/60/files

Reply:
I guess RoleCareTeamMember should probably change to RoleCarePartner as well.